File: go-client-sdk/pkg/models/shared/codatpullmessagescontractsfetchmessage.go
package shared

import (
	"time"
)

type CodatPullMessagesContractsFetchMessage struct {
	LogDateTimeUtc *time.Time `json:"logDateTimeUtc,omitempty"`
	LogLevel       *string    `json:"logLevel,omitempty"`
	Message        *string    `json:"message,omitempty"`
}

File: typescript-client-sdk/src/sdk/models/shared/codatpullmessagescontractsfetchmessage.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class CodatPullMessagesContractsFetchMessage extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=logDateTimeUtc" })
  logDateTimeUtc?: Date;

  @SpeakeasyMetadata({ data: "json, name=logLevel" })
  logLevel?: string;

  @SpeakeasyMetadata({ data: "json, name=message" })
  message?: string;
}
